typed_python/PyClassInstance.cpp

```diff
@@ -332,6 +347,16 @@ void PyClassInstance::constructFromPythonArgumentsConcrete(Class* classT, uint8_
 }
 
 PyObject* PyClassInstance::tp_getattr_concrete(PyObject* pyAttrName, const char* attrName) {
+    auto p = callMemberFunction("__getattribute__", pyAttrName);
```

Comment:
we need a `mHasGetAttributeMagicMethod` flag on the class when it's defined. Otherwise we have to do an extra lookup in a std::map each time we access a class attribute...

Reply:
Ok, this has been done.